tutorials/Bayesian_Logistic_Regression.ipynb

```diff
@@ -9,8 +9,6 @@
    "source": [
```

Comment:
<p>I think this is mistakenly being rendered as a sub heading?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/facebookresearch/beanmachine/pull/1250/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
Actually I don't see this on github, so maybe this is just a ReviewNB issue. Weird.